### DISCUSSION TITLE
Add Extentions

### DISCUSSION THREAD

**Discussion by VenexGit** [bromite/bromite#1379] [Q&A]
I think the browser would be better if the extensions feature was added.

**Comment by chromer030** [comment]
https://github.com/bromite/bromite/blob/master/FAQ.md#does-bromite-support-extensions

**Comment by chromer030** [comment]
Extension support for android is a downstream work and Bromite developers have not any plan to maintain it , there was an project named : [ungoogled-chromium-android](https://github.com/wchen342/ungoogled-chromium-android/releases) with support of extensions but due to very high amount of work which should be done and maintain in downstream , developer has stopped it.
Extension support could be possible if upstream add support for it.

**Reply by VenexGit** [reply]
I hope this happens. Thank you for the work you are doing to develop the browser. For me, it is the best browser I have used.
